fc/fc_app.py

```diff
@@ -206,6 +206,22 @@ def list_configs():
         print(cfg.name)
 
 
+@cli.command(help='Add FC product definitions to the datacube')
+@click.argument(
+    'app-config-files',
+    nargs=-1,
+    type=click.Path(exists=True, readable=True, writable=False, dir_okay=False)
+)
+@ui.config_option
+@ui.verbose_option
+@ui.pass_index(app_name=APP_NAME)
+def load_config(index, app_config_files):
```

Comment:
Awesome command! I'm going to add something similar to stats!

Not convinced by the name though, but won't break anything if we change it. ;-)